File: Experiment Processing/database/consent_form.py
from mongoengine import Document, ObjectIdField, StringField, DictField, DateTimeField


class ConsentForm(Document):
    """
    Model class for the consent forms in the database.
    """

    meta = {"collection": "consent_forms"}

    _id = ObjectIdField(required=True)

    email_address = StringField(required=True)
    session_id = StringField()

    consent_form = DictField(required=True)

    updated_at = DateTimeField()
    created_at = DateTimeField()

File: Web Survey/app/ConsentForm.php
<?php


namespace App;


use Jenssegers\Mongodb\Eloquent\Model;

/**
 * Class ConsentForm
 *
 * Model for the consent_forms table in the database.
 *
 * @package App
 */
class ConsentForm extends Model
{
    protected $primaryKey = "email_address";

    protected $fillable = [
        "email_address", "session_id", "consent_form"
    ];
}
